src/MunicipalityRegistry.Api.Extract/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM microsoft/dotnet:2.2.0-runtime-deps
+FROM microsoft/dotnet:2.2.4-runtime-deps
```

Comment:
This image doesn't exist, Microsoft refactored their entire Docker strategy, please use:

`FROM mcr.microsoft.com/dotnet/core/runtime-deps:2.2.4`